pkg/cloud/openstack/deployer.go

```diff
@@ -72,7 +72,7 @@ func (d *DeploymentClient) GetKubeConfig(cluster *clusterv1.Cluster, master *clu
 		return "", err
 	}
 
-	result := strings.TrimSpace(util.ExecCommand(
+	result := strings.TrimSpace(execCommand(
```

Comment:
I am not against this kind of change but seems we should leverage Utils function as much as we can
so is it possible to have a default log level (error) and use log.warn for error cases in this typical secenia?